packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -35,6 +35,7 @@ import '../../src/testbed.dart';
 const String _kNoPlatformsMessage = 'You\'ve created a plugin project that doesn\'t yet support any platforms.\n';
 const String frameworkRevision = '12345678';
 const String frameworkChannel = 'omega';
+const String _kDisabledPlatformRequestedMessage = 'not currently supported on your local environment.';
```

Comment:
The desktop message says "not supported", but the web message says "not currently supported"
Let's unify them for easier testing.
(The tests for desktop probably fail now)